Sources/Amplitude/Amplitude.m

```diff
@@ -780,21 +711,56 @@ - (void)uploadEventsWithLimit:(int)limit {
             return;
         }
 
-        long eventCount = [self.dbHelper getTotalEventCount];
+        long eventCount = [self->_eventsBuffer count];
```

Comment:
Should this be long or NSUInteger?

Reply:
they're all interchangeable, as long as we have no warnings complaining about lossy conversion. We can set a standard for this

Reply:
@dantetam  is correct. NSUInteger is just a typedef. @yuhao900914